pkgs/applications/networking/cluster/tanka/default.nix

```
@@ -15,6 +15,13 @@ buildGoModule rec {
 
   buildFlagsArray = [ "-ldflags=-s -w -X main.Version=${version}" ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    echo "complete -C $out/bin/tk tk" > $TMP/tk.bash
+    installShellCompletion $TMP/tk.bash
```

If anything `$TMPDIR`. But the derivation is already built in a temporary directory, so I'd rather use:

```suggestion
    echo "complete -C $out/bin/tk tk" > tk.bash
    installShellCompletion tk.bash
```